articles/110_interface_definition.md

```diff
@@ -70,11 +70,15 @@ The following primitive types are defined:
 * `string`
 
 <div class="alert alert-warning" markdown="1">
+  **Question:** in ROS 1, `string` was ASCII. The Python 3 `str` type is UTF-8. What is the DDS `string` type?
```

Comment:
Yes, I guess that's true, but not very satisfying as a specification.

Should we recommend these bytes contain either ASCII or a UTF-8 encoding?